app/views/shared/_navbar.html.erb

```diff
@@ -29,8 +29,8 @@
     </div>
 
     <!-- Button (call-to-action) -->
-    <a href="" class="navbar-wagon-item navbar-wagon-button btn">Post stuff</a>
-  </div>
+    <%= link_to "Add a watch", new_watch_path, class: "navbar-wagon-item navbar-wagon-button btn" %>
```

Comment:
@ArianePhelizot peux-tu juste placer le bouton "Add a watch avant la div `<!-- Profile picture and dropdown -->` ?